NuRadioReco/framework/event.py

```diff
@@ -15,8 +16,45 @@ def __init__(self, run_number, event_id):
         self._parameters = {}
         self.__run_number = run_number
         self._id = event_id
-        self.__stations = {}
+        self.__stations = collections.OrderedDict()
         self.__event_time = 0
+        self.__modules_event = []  # saves which modules were executed with what parameters on event level
+        self.__modules_station = {}  # saves which modules were executed with what parameters on station level
+
+    def register_module_event(self, instance, name, kwargs):
+        """
+        registers modules applied to this event
```

Comment:
There is documentation now, but that the register_module_event registers the modules applied to an event is kind of obvious :) Can you describe the parameter please?

Reply:
Sorry, I didn't mark these earlier.

Reply:
fixed